src/Symfony/Component/Translation/Command/XliffLintCommand.php

```diff
@@ -115,14 +117,16 @@ private function validate($content, $file = null)
         $document->loadXML($content);
 
         if (null !== $targetLanguage = $this->getTargetLanguageFromFile($document)) {
-            $expectedFileExtension = sprintf('%s.xlf', str_replace('-', '_', $targetLanguage));
-            $realFileExtension = explode('.', basename($file), 2)[1] ?? '';
+            $normalizedLocale = preg_quote(str_replace('-', '_', $targetLanguage));
+            // when this command is used inside Symfony, require filenames to be '____.locale.xlf'
+            // when used outside Symfony, allow '____.locale.xlf' and 'locale.____.xlf'
```

Comment:
comments should be talking about strict vs not strict, not about `inside Symfony`. Other projects may put the command in strict mode too.